india_compliance/gst_india/overrides/supplier.py

```diff
@@ -22,8 +22,16 @@ def update_transporter_gstin(doc, method=None):
                 ).format(bold(doc.gstin)),
                 alert=True,
             )
+            return
 
-        return
+        if doc.pan != doc.gst_transporter_id[2:12]:
+            frappe.throw(
+                _(
+                    "The PAN of GSTIN {0} doesn't match the PAN of the GST Transporter"
```

Comment:
let's be accurate please -> just say "PAN of transporter" instead of "PAN of gstin". what if pan doesn't match because user has customised behavior?